UnitystationLauncher/Views/NewsView.xaml

```diff
@@ -8,9 +8,26 @@
   <ItemsControl Items="{Binding PullRequests}">
     <ItemsControl.ItemTemplate>
       <DataTemplate>
-        <Button Command="{Binding VisitURL}" Margin="2" BorderThickness="1" BorderBrush="Gray" Background="#222D3C" MaxWidth="180" MinHeight="40" Cursor="Hand">
-          <TextBlock Text="{Binding Title}" TextWrapping="Wrap" VerticalAlignment="Center" TextAlignment="Center" FontSize="12" Margin="1"/>
-        </Button>
+        <ItemsControl Background="#dddddd" BorderBrush="#3377a0" BorderThickness="0.5" MaxHeight="500" Margin="5 5 5 5" Padding="4 4 4 4">
```

Comment:
`Background="#dddddd"`
This doesn't fit in with the rest of the UI, choose some darker color for the background and a lighter one for the text.